cmd/mistryd/server.go

```diff
@@ -350,19 +352,29 @@ func (s *Server) ListenAndServe() error {
 
 // RebuildResult contains result data on the rebuild operation
 type RebuildResult struct {
-	successful  int
-	failed      int
-	pruneReport dockerTypes.ImagesPruneReport
+	successful       int
+	failed           []string
+	prunedImages     int
+	prunedContainers int
+	reclaimedSpace   uint64
 }
 
 func (r RebuildResult) String() string {
+	var failedNames string
+	if len(r.failed) > 0 {
+		failedNames = " Failed names: " + strings.Join(r.failed, ", ")
+	} else {
+		failedNames = ""
+	}
+
 	return fmt.Sprintf(
-		"Rebuilt: %d Failed: %d Pruned: %d Reclaimed: %d bytes",
-		r.successful, r.failed, len(r.pruneReport.ImagesDeleted), r.pruneReport.SpaceReclaimed)
+		"Rebuilt: %d Pruned images: %d Pruned containers: %d Reclaimed: %s Failed: %d%s",
+		r.successful, r.prunedImages, r.prunedContainers, units.HumanSize(float64(r.reclaimedSpace)),
+		len(r.failed), failedNames)
 }
 
 // RebuildImages rebuilds images for all projects, and prunes any dangling images
-func RebuildImages(cfg *Config, log *log.Logger, out io.Writer, projects []string, failOnImageError bool) (RebuildResult, error) {
+func RebuildImages(cfg *Config, log *log.Logger, projects []string, failOnImageError, verbose bool) (RebuildResult, error) {
```

Comment:
[nit] Maybe `stopErr` would be a better name for that variable. The function does not fail, it returns immediately.